aspnetcore/signalr/scale.md

```diff
@@ -15,6 +15,18 @@ By [Andrew Stanton-Nurse](https://twitter.com/anurse), [Brady Gaster](https://tw
 
 This article explains hosting and scaling considerations for high-traffic apps that use ASP.NET Core SignalR.
 
+## Sticky Sessions
+
+SignalR requires that all HTTP requests for a specific connection be handled by the same server process. This means that "sticky sessions" (also called "session affinity" by some load balancers or "ARR affinity" in Azure App Service) are required in order to use SignalR. The **only** circumstances in which sticky sessions are **not** required are:
```

Comment:
SignalR requires that all HTTP requests for a specific connection be handled by the same server process. When SignalR is running on a server farm (multiple servers), "sticky sessions" must be used. "Sticky sessions" are also called session affinity by some load balancers. Azure App Service use [Application Request Routing](https://www.iis.net/learn/extensions/planning-for-arr) (ARR) to route all subsequent requests to the same server. The **only** circumstances in which sticky sessions are **not** required are:

Reply:
For some reason it's not offering to commit this suggestion. Let me know if you'd like me to do that and S&M (squash and merge).

Reply:
That's still not quite right. ARR **doesn't** provide affinity by default, it *has to be enabled*. How about something like the below content. If you're good with it, go ahead and apply it and squash and merge.

A second option would be to just link to our existing doc content on configuring Azure App Service (https://docs.microsoft.com/en-us/aspnet/core/signalr/publish-to-azure-web-app?view=aspnetcore-2.2#configure-the-app-in-azure-app-service). I do that in the final sentence of this change but we could move that up here. That may be better than trying to explain all of this in a single paragraph (without run-on sentences or overloading the paragraph)

---

SignalR requires that all HTTP requests for a specific connection be handled by the same server process. When SignalR is running on a server farm (multiple servers), "sticky sessions" must be used. "Sticky sessions" are also called session affinity by some load balancers. Azure App Service uses [Application Request Routing](https://docs.microsoft.com/iis/extensions/planning-for-arr/application-request-routing-version-2-overview) (ARR) to route requests. Enabling the "ARR Affinity" setting in your Azure App Service will enable "sticky sessions". The only circumstances in which sticky sessions are not required are: